vtkext/private/module/vtkF3DImguiActor.cxx

```diff
@@ -50,6 +53,26 @@ struct vtkF3DImguiActor::Internals
       // Create VBO
       this->VertexBuffer = vtkSmartPointer<vtkOpenGLBufferObject>::New();
 
+      // Load embedded PNG icon into texture
+      vtkNew<vtkPNGReader> iconReader;
+      iconReader->SetMemoryBuffer(F3DDefaultLogo);
+      iconReader->SetMemoryBufferLength(sizeof(F3DDefaultLogo));
+      iconReader->Update();
+
+      vtkImageData* imageData = iconReader->GetOutput();
+      int dims[3];
+      imageData->GetDimensions(dims);
```

Comment:
```suggestion
      int dims[3] = imageData->GetDimensions();
```

Reply:
This comment is not addressed. I think this is the correct code though:
```cpp
int* dims = imageData->GetDimensions();
```